### ISSUE TITLE
Give "lock wheelsteering" actual PID control

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2106]
(This has been true for some time, thus why I'm not calling it a bug report although  it sort of almost is.)

``Lock Wheelsteering to val.`` has three uses depending on the value type:
1 - lock it to another vessel.
2 - lock it to a geocoordinates.
3 - lock it to a scalar, which is a compass reading to lock onto.

For all three of these, they end up eventually doing the same thing - seeking a bearing of zero to the target.
(The logic that causes all of them to eventually reduce to just the same thing is in ``WheelSteeringManager.UpdateValue()``.)

And if you look at how it works, they basically just set the FlightCtrlState.wheelSteer`` to a value that is 1/10th of the current bearing offset.  That's just a classic P-only controller, with a Kp of 0.1, with no derivative or integral term.

And in practice if you try using it for a while you can see that ``lock wheelsteering to some_heading.`` has all the same classic problems that a P-only controller has.  While it might be possible to pick a Kp that makes it work okay for some particular rover design, the player has zero control over that Kp term.  (the divide by 10 is hardcoded).  So ``lock wheelsteering` ends up being nowhere near a generic solution for all rovers.  Not even close.  It only works for a rover going about 1 or 2 m/s.  Higher than that and it starts waggling back and forth all over the place, which would cause a flipover in a low gravity situation.

Perhaps an argument could be made that the steering shouldn't be perfect and the player should have to work at it a bit, but they should at least be provided with the tools to do so without having to resort to raw control (which is what I did in my rover driving script ages ago).

In other words, instead of making it a hardcoded P-only contoller with a hard-code Kp of 0.1, we should allow the script to tweak its settings.  Make it a real PID controller, default its Kp to 0.1, Kd to 0, and Ki to 0, and it should behave exactly as (badly) as it does now.  But then give the user some kind of handle to alter those Kp, Ki, and Kd terms.
